packages/react/src/Delay.tsx

```diff
@@ -1,17 +1,33 @@
-import { type PropsWithChildren, useContext, useState } from 'react'
+import { type PropsWithChildren, type ReactNode, useContext, useState } from 'react'
 import { DelayDefaultPropsContext } from './contexts'
 import { useTimeout } from './hooks'
+import { assert } from './utils'
 
 export interface DelayProps extends PropsWithChildren {
   ms?: number
+  /**
+   * @experimental This is experimental feature.
+   */
+  fallback?: ReactNode
 }
 
-export const Delay = ({ ms, children }: DelayProps) => {
+export const Delay = (props: DelayProps) => {
   const defaultProps = useContext(DelayDefaultPropsContext)
-  const delayMs = ms ?? defaultProps.ms ?? 0
-  const [isDelayed, setIsDelayed] = useState(delayMs === 0)
-  useTimeout(() => setIsDelayed(true), delayMs)
-  return <>{isDelayed ? children : null}</>
+
+  const ms = props.ms ?? defaultProps.ms ?? 0
+  if (process.env.NODE_ENV !== 'production') {
+    assert(
+      [props.ms, defaultProps.ms].every((item = 0) => item >= 0),
+      'Delay: ms prop should be greater than or equal to 0'
+    )
+  }
```

Comment:
@minsoo-web I added assertion for `ms` prop in development mode.

Reply:
```suggestion
  if (process.env.NODE_ENV !== 'production') {
    assert(ms > 0, 'Delay: ms prop should be greater than or equal to 0')
  }
```

이렇게 해도 되지 않아?? 바로 윗 줄에서 `ms`를 `nullish coalescing operator` (??) 을 통해 제어했으니깐 똑같은 처리라고 생각해